python/tvm/rpc/client.py

```diff
@@ -378,7 +402,7 @@ def request_and_run(self,
                 key, max_retry, str(last_err)))
 
 
-def connect(url, port, key="", session_timeout=0):
+def connect(url, port, key="", session_timeout=0, session_constructor=None):
```

Comment:
maybe we can call it session_constructor_args so it's more clear it should be a list rather than a callable?